packages/sanity/src/core/releases/tool/detail/__tests__/ReleaseSummary.test.tsx

```diff
@@ -1,115 +1,95 @@
-import {defineType} from '@sanity/types'
-import {fireEvent, render, screen, within} from '@testing-library/react'
+import {act, fireEvent, render, screen, within} from '@testing-library/react'
 import {route, RouterProvider} from 'sanity/router'
 import {beforeEach, describe, expect, it, vi} from 'vitest'
 
-import {getAllByDataUi, getByDataUi} from '../../../../../../test/setup/customQueries'
+import {getByDataUi} from '../../../../../../test/setup/customQueries'
 import {createTestProvider} from '../../../../../../test/testUtils/TestProvider'
+import {DefaultPreview} from '../../../../components/previews/general/DefaultPreview'
+import {
+  activeASAPRelease,
+  archivedScheduledRelease,
+  scheduledRelease,
+} from '../../../__fixtures__/release.fixture'
 import {releasesUsEnglishLocaleBundle} from '../../../i18n'
-import {type ReleaseDocument} from '../../../index'
-import {type DocumentHistory} from '../documentTable/useReleaseHistory'
 import {ReleaseSummary, type ReleaseSummaryProps} from '../ReleaseSummary'
 import {type DocumentInRelease} from '../useBundleDocuments'
-
-vi.mock('../../../../studio/addonDataset/useAddonDataset', () => ({
-  useAddonDataset: vi.fn().mockReturnValue({client: {}}),
+import {
+  documentsInRelease,
+  useBundleDocumentsMockReturnWithResults,
+} from './__mocks__/useBundleDocuments.mock'
+
+vi.mock('../../../index', () => ({
+  useDocumentPresence: vi.fn().mockReturnValue({
+    user: '',
+    path: '',
+    sessionId: '',
+    lastActiveAt: '',
+  }),
+  useDocumentPreviewStore: vi.fn().mockReturnValue({
+    unstable_observeDocumentIdSet: vi.fn(() => ({
+      pipe: vi.fn(),
```

Comment:
Could  the `unstable_observeDocumentIdSet` mock instead return an observable? It's not clear to me what mocking `pipe()` here does 🤔